### ISSUE TITLE
Slider scrolls images only after changing the width or height of the browser

### ISSUE THREAD

**Issue by AlexeyDanilov** [nolimits4web/Swiper#2254]
Hallo! Faced a problem - implemented the Swiper slider, as always. But now in no browser when clicking on the arrows, it does not start, in the inspector you can see that the script works. The slider itself switches only after changing the size of the browser and then it works normally. I have never encountered such a problem before. Just in case, the slider is implemented in a tab.

`
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Отделочные работы</title>
	<link rel="stylesheet" href="styles.css">
	<link rel="stylesheet" type="text/css" href="css/normalize.css" />

	<link rel="stylesheet" type="text/css" href="css/style-ama.css" />
	<link rel="stylesheet" type="text/css" href="pater/pater.css" />
	<link href="https://fonts.googleapis.com/css?family=Pangolin&amp;subset=cyrillic,cyrillic-ext,latin-ext,vietnamese" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
	 <link rel="stylesheet" href="dist/css/swiper.min.css">

</head>
<body>
<div id="tabs-container">
	<div class="container">
	    <div class="tabs-menu menu menu--ama">
	        <a class="current menu__item" href="#tab-1">
	        <span class="menu__item-name">Наши услуги</span>
	        </a>
	        <a class="menu__item" href="#tab-2">
	        <span class="menu__item-name">Наши клиенты</span>
	        </a>
	        <a class="menu__item" href="#tab-3">
	        <span class="menu__item-name">Контакты</span>
	        </a>
	    </div>
	    <div class="tab">
	        <div id="tab-1" class="tab-content">
	        	<ol class="listServices rectangle-list">
	        		<li class="listServices__item">
	        			<div class="short">Гипсокартонные работы<img src="img/arrow.png" alt="" class="arrow"></div>
	        			<p class="detail">стены, потолки, короб, перегородки, ниши, арки</p>
	        		</li>

	        		<li class="listServices__item">
	        			<div class="short">Укладка напольных покрытий<img src="img/arrow.png" alt="" class="arrow"></div>
						<p class="detail">линолеум, ламинат, ковролин</p>
	        		</li>

	        		<li class="listServices__item">
	        			<div class="short">Потолки<img src="img/arrow.png" alt="" class="arrow"></div>
						<p class="detail">армстронг, ПВХ, ГКЛ</p>			
	        		</li>

	        		<li class="listServices__item">
	        			<div class="short">Отделочные работы<img src="img/arrow.png" alt="" class="arrow"></div>
						<p class="detail">укладка декоративной плитки, шпаклёвка стен, покраска, поклейка обоев</p>	
	        		</li>

	        		<li class="listServices__item">
	        			<div class="short">Электромонтаж</div>
	        		</li>

	        		<li class="listServices__item">
	        			<div class="short">Сантехника</div>
	        		</li>
	        	</ol>
	        </div>
	        <div id="tab-2" class="tab-content">
	        	<div class="clients">
	        		<div class="clients__item">
	        			<a href="http://cafe-sady.ru/" class="clients-link" target="_blank">
	        				<img src="img/sady.jpg" alt="" class="clients__item-logo">
	        			</a>
	        		</div>

	        		<div class="clients__item">
	        			<a href="http://mirinkub.ru/" class="clients-link" target="_blank">
	        				<img src="img/mirinkub.png" alt="" class="clients__item-logo">
	        			</a>
	        		</div>

	        		<div class="clients__item">
	        			<a href="http://sedmoy-pol.ru/" class="clients-link" target="_blank">
	        				<img src="img/pol.jpg" alt="" class="clients__item-logo">
	        			</a>
	        		</div>
	        	</div>

	        	<div class="swiper-container jobs-container">
				    <div class="swiper-wrapper">
					    <div class="swiper-slide"><img src="img/jobs/1.jpg" alt=""></div>
					    <div class="swiper-slide"><img src="img/jobs/2.jpg" alt=""></div>
					    <div class="swiper-slide"><img src="img/jobs/3.jpg" alt=""></div>
					    <div class="swiper-slide"><img src="img/jobs/4.jpg" alt=""></div>
					    <div class="swiper-slide"><img src="img/jobs/5.jpg" alt=""></div>
				    </div>
			
				    <div class="swiper-button-next"></div>
				    <div class="swiper-button-prev"></div>
 				</div>

	        
	        </div>
	        <div id="tab-3" class="tab-content">
	        	<div class="contact__wraper">
		        	<div class="phone__wraper">
		        		<div class="phone-img__wraper"> 	
			        		<img class="phone-pic" src="img/phone.png" alt="">
			        		<img src="img/WhatsApp.png" alt="">
		        		</div>
		        		<span class="phoneNumber">+7(927)403-68-32
							<span class="city">Республика Татарстан г. Казань</span>
		        		</span>	
		        	</div>
		        	<a class="vk__link" href="https://vk.com" target="_blank"><img class="vk-icon" src="img/vk.png" alt=""></a>
		        </div>
	        </div>

	    </div>
	</div>
</div>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="script.js"></script>
	<script>document.documentElement.className="js";var supportsCssVars=function(){var e,t=document.createElement("style");return t.innerHTML="root: { --tmp-var: bold; }",document.head.appendChild(t),e=!!(window.CSS&&window.CSS.supports&&window.CSS.supports("font-weight","var(--tmp-var)")),t.parentNode.removeChild(t),e};supportsCssVars()||alert("Please view this demo in a modern browser that supports CSS Variables.");</script>	
		<script src="js/charming.min.js"></script>
		<script src="js/anime.min.js"></script>
		<script src="js/demo.js"></script>
		<script src="js/demo-ama.js"></script>
		<script src="dist/js/swiper.jquery.min.js"></script>
		<script src="dist/js/swiper.min.js"></script>
		<script src="slider.js"></script>
		<script src="dist/js/swiper.js"></script>
</body>
</html>

`

`

body{
    padding: 0;
    margin: 0;
}

#tabs-container{
    width: 100%;
    // background-image: url('img/escheresque_ste.png');
    background-image: linear-gradient(to right, rgba(0, 0, 255, 0.45), rgba(0, 0, 255, 0.45)), url('img/bg.jpg');
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
    height: 1000px;
}

.container{
    width: 80%;
    margin: 0 auto;
}

.tabs-menu {
    display: flex;
    justify-content: space-around;
    align-items: center;
    margin-top: -1px;
    padding-top: 15px;
}

.tabs-menu li {
    height: 30px;
    line-height: 30px;
    float: left;
    margin-right: 10px;
    background-color: #ccc;
    border-top: 1px solid #d4d4d1;
    border-right: 1px solid #d4d4d1;
    border-left: 1px solid #d4d4d1;
    list-style-type: none;
}

.tabs-menu li.current {
    position: relative;
    background-color: tra;
    border-bottom: 1px solid #fff;
    z-index: 5;
}

.tabs-menu li a {
    padding: 10px;
    text-transform: uppercase;
    color: #fff;
    text-decoration: none; 
}

.tabs-menu .current a {
    color: #2e7da3;
}

.tab {
    width: 100%;
}

.tab-content {
    width: 100%;
    text-align: center;
    justify-content: center;
    display: none;
}

.listServices__item{
    font-family: 'Montserrat', sans-serif;
    color: white;
    list-style-type: none;
    display: flex;
    flex-direction: column;
}

#tab-1 {
 display: block;   
}

.rectangle-list{
    counter-reset: li;
}

.rectangle-list li {
    position: relative;
    display: block;
    padding: .4em .4em .4em .8em;
    *padding: .4em;
    margin: .5em 0 .5em 2.5em;
    background: #ddd;
    color: #444;
    text-decoration: none;
    -webkit-transition: all .3s ease-out;
    -moz-transition: all .3s ease-out;
    -ms-transition: all .3s ease-out;
    -o-transition: all .3s ease-out;
    transition: all .3s ease-out;   
}
 
.rectangle-list li:hover {
    background: #42f4aa;
}   
 
.rectangle-list li:before {
    content: counter(li);
    counter-increment: li;
    position: absolute; 
    left: -2.5em;
    top: 50%;
    margin-top: -1em;
    background: #fa8072;
    height: 2em;
    width: 2em;
    line-height: 2em;
    text-align: center;
    font-weight: bold;
}
 
.rectangle-list li:after {
    position: absolute; 
    content: '';
    border: .5em solid transparent;
    left: -1em;
    top: 50%;
    margin-top: -.5em;
    -webkit-transition: all .3s ease-out;
    -moz-transition: all .3s ease-out;
    -ms-transition: all .3s ease-out;
    -o-transition: all .3s ease-out;
    transition: all .3s ease-out;               
}
 
.rectangle-list li:hover:after {
    left: -.5em;
    border-left-color: #fa8072;             
}

.phone-pic{
    width: 25px;
    height: auto;
    margin-bottom: 15px;
}

.vk-icon{
    width: 25px;
    height: auto;
}

.vk__link{
    margin-top: 15px;
}

.contact__wraper{
    width: 30%;
    margin: 0 auto;
    display: flex;
    flex-direction: column;
    align-items: flex-start;
}

.phone__wraper{
    width: 100%;
    margin: 0 auto;
    display: flex;
    justify-content: space-between;
    align-items: center;
}

.phone-img__wraper{
    display: flex;
    flex-direction: column;
    align-items: center;
}

.phoneNumber{
    font-family: 'Montserrat', sans-serif;
    color: white;
    font-weight: 600;
    font-size: 25px;
}

.city{
    font-family: 'Montserrat', sans-serif;
    color: white;
    font-weight: 600;
    font-size: 14px; 
}

.short{
    position: relative;
    cursor: pointer;
}

.detail{
    display: none;
    color: black;
    font-weight: bold;
}

.arrow{
    position: absolute;
    right: 19px;
    top: 5px;
    width: 10px;
    height: auto;
}

.rotate{
    transform: rotate(180deg);
}

.clients{
    display: flex;
    justify-content: space-between;
    align-items: center;
    flex-wrap: wrap;
}

.clients__item{
    width: 25%;
}

.clients-link{
    width: 100%;
}

.clients__item-logo{
    width: 100%;
    height: auto;
}

.jobs-container{
    max-width: 500px;
    height: auto;
    margin: 0 auto;
    margin-top: 100px;
}

// .swiper-wrapper{
//     width: 100%;
//     height: auto;
// }

// .swiper-slide{
//     width: 100%;
//     height: auto;
// }

.swiper-slide img{
    max-width: 100%;
    height: auto;
}

`

`
$(document).ready(function(){
  
 
    $(".tabs-menu a").click(function(event) {
        event.preventDefault();
        $(this).parent().addClass("current");
        $(this).parent().siblings().removeClass("current");
        var tab = $(this).attr("href");
        $(".tab-content").not(tab).css("display", "none");
        $(tab).fadeIn();
    });
 
 
 
    $(".short").click(function(){
    $(this).find(':first-child').toggleClass('rotate')
    $(this).next().slideToggle("slow");
    $(this).toggleClass("active"); return false;
    });
 
    
  
    var swiper = new Swiper('.jobs-container', {
        spaceBetween: 30,    
        speed: 2000,
        nextButton:'.swiper-button-next',
        prevButton:'.swiper-button-prev'
    });
});
`

**Comment by AlexeyDanilov** [comment]
The project is in my repository with the name Ilnur

**Comment by Dizzzy** [comment]
Make sure the content doesn't have display: hidden when 
Swiper initialises. 

$(".tab-content").not(tab).css("display", "none"); - could this be it? Also display: none isn't valid CSS

**Comment by AlexeyDanilov** [comment]
Thank you! Now it works.

`    $(".tabs-menu a").click(function(event) {
        event.preventDefault();
        $(this).parent().addClass("current");
        $(this).parent().siblings().removeClass("current");
        var tab = $(this).attr("href");
        $(".tab-content").not(tab).css("display", "none");
        $(tab).fadeIn();


    var swiper = new Swiper('.jobs-container', {
    spaceBetween: 30,    
    speed: 2000,
      navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
      },
    });

    });



	$(".short").click(function(){
	$(this).find(':first-child').toggleClass('rotate')
	$(this).next().slideToggle("slow");
	$(this).toggleClass("active"); return false;
	});`

**Comment by nolimits4web** [comment]
Looks like it is fixed
